src/routes/microscope/operations/microscope-dlq-operations.ts

```diff
@@ -0,0 +1,88 @@
+import { Request, Response } from "express";
+import Logger from "bunyan";
+import { getLogger } from "config/logger";
+import { optionalRequire } from "optional-require";
+
+const { DlqService } = optionalRequire("@atlassian/sqs-queue-dlq-service", true) || {};
+
+const log: Logger = getLogger("microscope-dlq");
+
+type RequeueMessageRequest = {
+	messageId: string;
+	receiptHandle: string;
+	body: string;
+	messageAttributes: object;
+};
+
+type DeleteMessageRequest = {
+	receiptHandle: string;
+};
+
+let dlqServiceClient;
+
+if (!DlqService) {
+	log.info("No dlqService package - skipping dlq operations");
+} else {
+	dlqServiceClient = new DlqService(log, {
+		sqs: {
+			region: process.env.SQS_PUSH_QUEUE_REGION
+		}
+	});
+}
+
+export const microscopeDlqHealthcheck = async (_: Request, res: Response): Promise<void> => {
```

Comment:
Think we should add tests for these middlewares?

Reply:
Sooo testing those are not as straight forward because of the optional package (we can't just mock it). As I mentioned on the PR description, I haven't added a UT but the endpoint is protected by POCO and can only be called via Microscope. I do intend to go back and try to find a way to test those but had a chat with Sass and this has been going on for too long so it shouldn't be a blocker.